CONTRIBUTING.md

```diff
@@ -62,3 +62,27 @@ We use [SassDoc] to document Bourbon. [Annotations] should be ordered:
 
 [SassDoc]: http://sassdoc.com
 [Annotations]: http://sassdoc.com/annotations
+
+## Goverance
+
+We, the maintainers of Bourbon, would like you to know what to expect when
+interacting with this project's repo.
+
+As a user of Bourbon, you can expect care to be put into official releases. The
+maintainers cannot guarantee that the `master` branch will have the same API
+forever.
+
+As someone who is reviewing pull requests or commenting on issues, you are
+expected to abide by our [code of conduct].
+
+As a contributor submitting a pull request or opening an issue, you can expect
```

Comment:
From a formatting standpoint, what do you think about bolding the main types of users in each paragraph (so bolding `contributor` here)? So when someone is skimming the content, it's easy to find what's relevant to them.